iree/compiler/Conversion/LinalgToLLVM/KernelDispatch.cpp

```diff
@@ -129,35 +151,39 @@ DEFINE_TILE_SIZE_FN(TilingLevel::Level2Tiles)
 
 #undef DEFINE_TILE_SIZE_FN
 
+bool isDispatchOp(Operation *op) {
+  if (auto contractionOp = dyn_cast<linalg::ContractionOpInterface>(op)) {
+    if (contractionOp.isRowMajorMatmul() ||
+        contractionOp.isRowMajorBatchMatmul()) {
+      return true;
+    }
+  }
+  if (isa<linalg::GenericOp>(op)) return true;
```

Comment:
What happens with indexed_generic ? We can stage this. The GPU side should be able to handle `IndexedGeneric` too, its just not plumbed through this path on the dispatch region creation step.

Reply:
we can do that in a follow up PR, need to enable it in `DispatchLinalg..` pass too